tests/samples/yaml/epics.ibek.yaml

```diff
@@ -0,0 +1,31 @@
+# yaml-language-server: $schema=file://schemas/ibek.schema.json
```

Comment:
I like epics.ibek.yaml.

There are probably other things that would go in here I'm guessing.